src/models/category.ts

```diff
@@ -0,0 +1,10 @@
+export class Category{
+categoryId: number;
+categoryName: string;
+
```

Comment:
Are you sure you want them to be called categoryId and categoryName? instead of just id and name?
the difference is category.categoryId vs category.id right.
Review with batch about these names